scss/_variables.scss

```diff
@@ -240,6 +240,7 @@ $font-size-lg:   1.25rem !default;
 $font-size-sm:   .875rem !default;
 $font-size-xs:   .75rem !default;
 
+$font-weight-light: light !default;
```

Comment:
There's no such thing as `font-weight: light` - there's `lighter`, but it's relative and I'm not sure it's what you wanted to achieve here. Alternatively it could be `300`, but it would need to be tested with the font stack if it reliably supports this weight (or at least does a fallback to other weight that 'looks light')